<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Телефонная книжка</string>
  </property>
  <widget class="QLabel" name="contactName_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="contactNumber_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Телефон</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>9</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0, 0, 0, 0);
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="contactNumber">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>49</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0, 0, 0, 0);
border-radius: 10px;
border: 3px solid #b7b7b7;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addContactBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 3px solid #b7b7b7;
border-radius: 10px;
background-color: none;
}
QPushButton:hover{
background-color: #c8c8c8;
}</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="contactList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>311</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border-radius: 5px;
border: 3px solid #b7b7b7;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
